tests/postgres/mod.rs

```diff
@@ -46,24 +46,27 @@ async fn test() -> Result<()> {
     .await?;
 
     // Publish and validate a component package
-    run(&root, |config| async move {
-        let client = super::create_client(&config)?;
-        super::publish_component_package(&client).await?;
-        super::validate_component_package(&config, &client).await?;
+    let digest = run(&root, |config| async move {
+        let client: warg_client::Client<
```

Comment:
Weird, I must have accidentally hit a rust-analyzer tooltip to explicitly qualify this, which isn't necessary. I'll remove.